.github/CODEOWNERS

```diff
@@ -1980,7 +1981,8 @@ x-pack/platform/plugins/shared/ml/server/models/data_recognizer/modules/security
 /x-pack/platform/test/api_integration/services/usage_api.ts @elastic/kibana-core
 /x-pack/platform/test/api_integration/apis/kibana @elastic/kibana-core
 /src/platform/test/api_integration/fixtures/import.ndjson @elastic/kibana-core
-/x-pack/platform/test/plugin_api_integration @elastic/kibana-core # Assigned per https://github.com/elastic/kibana/pull/146704
+x-pack/platform/test/plugin_api_integration/test_suites/platform/elasticsearch_client.ts @elastic/kibana-core
+x-pack/platform/test/plugin_api_integration/plugins/elasticsearch_client @elastic/kibana-core
```

Comment:
instead of all of /x-pack/platform/test/plugin_api_integration ownership is now more targeted

Reply:
event_log wasn't covered, so added:
```
/x-pack/platform/test/plugin_api_integration/test_suites/event_log/ @elastic/response-ops
```